src/components/themed/WalletList.js

```diff
@@ -46,6 +59,51 @@ type DispatchProps = {
 type Props = OwnProps & StateProps & DispatchProps & ThemeProps
 
 class WalletListComponent extends React.PureComponent<Props> {
+  sortWalletList(walletList: WalletListItem[]) {
+    const getFiatBalance = (wallet: GuiWallet, fullCurrencyCode: string) => {
+      const { settings, exchangeRates } = this.props
+      const currencyCode = getSortOptionsCurrencyCode(fullCurrencyCode)
+      return parseFloat(calculateWalletFiatBalanceUsingDefaultIsoFiat(wallet, currencyCode, settings, exchangeRates))
```

Comment:
No need to parse float if the calculate wallet Fiat balance using default ISO Fiat simply returns a number instead of a string.

Reply:
fixed